### ISSUE TITLE
New Feature Request: Add "Students" resource category

### ISSUE THREAD

**Issue by fvcproductions** [techqueria/website#357]
## Request

Let's add a resource category for Latinx students

## URL(s)

https://techqueria.org/resources/students/

## Why did you make this request?

#356

## Any additional context?

This needs to be completed before #356 can be done

**Comment by sonali9696** [comment]
@fvcproductions Do you want just a page to be created and added to the navbar or to populate it also like the nonprofits page?

**Comment by fvcproductions** [comment]
@sonali9696 We can just add to navbar for now and create the page since #356 is the first item to populate the page 👍🏽

**Comment by co16353sidak** [comment]
Hey everyone, would love to work on this, if you are allowing outsiders to work in this issue please give me the go ahead. 😃 
@sonali9696 @fvcproductions

**Comment by fvcproductions** [comment]
@co16353sidak I will assign to @sonali9696 for now but feel free to grab another issue 👍

**Comment by sonali9696** [comment]
Any emoji in mind?

**Comment by sonali9696** [comment]
Any emoji in mind for the Students category?

**Comment by sonali9696** [comment]
Once you confirm the emoji, I will raise the PR. The rest of the changes are pushed.

**Comment by fvcproductions** [comment]
@sonali9696 The emoji can be 🎓 but should not be included on the navbar for consistency.

**Comment by sonali9696** [comment]
Alright done :)
